rosdep/oe-opkg.yaml

```diff
@@ -0,0 +1,335 @@
+
+cppcheck:
+  oe:
+    opkg:
+      packages: [cppcheck-native@meta-sca]
+
+curl:
+  oe:
+    opkg:
+      packages: [curl@openembedded-core]
+
+eigen:
+  oe:
+    opkg:
+      packages: [libeigen@meta-oe]
+
+git:
+  oe:
+    opkg:
+      packages: [git@openembedded-core]
+
+gtk3:
+  oe:
+    opkg:
+      packages: [gtk+3@openembedded-core]
+
+libcairo2-dev:
+  oe:
+    opkg:
+      packages: [cairo@openembedded-core]
+
+libceres-dev:
+  oe:
+    opkg:
+      packages: [ceres-solver@meta-oe]
+
+libconsole-bridge-dev:
+  oe:
+    opkg:
+      packages: [console-bridge@meta-ros]
+
+libexpat1-dev:
+  oe:
+    opkg:
+      packages: [expat@openembedded-core]
+
+libfreetype6-dev:
+  oe:
+    opkg:
+      packages: [freetype@openembedded-core]
+
+libfreetype6:
+  oe:
+    opkg:
+      packages: [freetype@openembedded-core]
+
+libgflags-dev:
+  oe:
+    opkg:
+      packages: [gflags@meta-oe]
+
+libglfw3-dev:
+  oe:
+    opkg:
+      packages: [glfw@meta-intel-realsense]
+
+libgpgme-dev:
+  oe:
+    opkg:
+      packages: [gpgme@openembedded-core]
+
+libogg:
+  oe:
+    opkg:
+      packages: [libogg@openembedded-core]
+
+libopencv-dev:
+  oe:
+    opkg:
+      packages: [opencv@meta-oe]
+
+libpcl-all-dev:
+  oe:
+    opkg:
+      packages: [pcl@meta-ros]
+
+libpcl-all:
+  oe:
+    opkg:
+      packages: [pcl@meta-ros]
+
+libpoco-dev:
+  oe:
+    opkg:
+      packages: [poco@meta-oe]
+
+libqt5-core:
+  oe:
+    opkg:
+      packages: [qtbase@meta-qt5]
+
+libqt5-gui:
+  oe:
+    opkg:
+      packages: [qtbase@meta-qt5]
+
+libqt5-opengl:
+  oe:
+    opkg:
+      packages: [qtbase@meta-qt5]
+
+libqt5-widgets:
+  oe:
+    opkg:
+      packages: [qtbase@meta-qt5]
+
+libsqlite3-dev:
+  oe:
+    opkg:
+      packages: [sqlite3@openembedded-core]
+
+libssl-dev:
+  oe:
+    opkg:
+      packages: [openssl@openembedded-core]
+
+libtheora:
+  oe:
+    opkg:
+      packages: [libtheora@openembedded-core]
+
+libudev-dev:
+  oe:
+    opkg:
+      packages: [eudev@openembedded-core]
+
+libusb-1.0-dev:
+  oe:
+    opkg:
+      packages: [libusb1@openembedded-core]
+
+libwebp-dev:
+  oe:
+    opkg:
+      packages: [libwebp@openembedded-core]
+
+libx11-dev:
+  oe:
+    opkg:
+      packages: [libx11@openembedded-core]
+
+libxaw:
+  oe:
+    opkg:
+      packages: [libxaw@meta-oe]
+
+libxml2-utils:
+  oe:
+    opkg:
+      packages: [libxml2@openembedded-core]
+
+libxrandr:
+  oe:
+    opkg:
+      packages: [libxrandr@openembedded-core]
+
+libzmq3-dev:
+  oe:
+    opkg:
+      packages: [zeromq@meta-oe]
+
+lua5.2-dev:
+  oe:
+    opkg:
+      packages: [lua@meta-oe]
+
+opengl:
+  oe:
+    opkg:
+      packages: [mesa@openembedded-core]
+
+openssl:
+  oe:
+    opkg:
+      packages: [openssl@openembedded-core]
+
+pcre:
+  oe:
+    opkg:
+      packages: [libpcre@openembedded-core]
+
+pkg-config:
+  oe:
+    opkg:
+      packages: [pkgconf@openembedded-core]
+
+protobuf-dev:
+  oe:
+    opkg:
+      packages: [protobuf@meta-oe]
+
+pyflakes3:
+ oe:
```

Comment:
The CI is failing due to an indentation error in this stanza.

Reply:
Thanks, that's a good catch here and below.